ui/components/MesherySettingsPerformanceComponent.js

```diff
@@ -111,7 +111,7 @@ class MesherySettingsPerformanceComponent extends React.Component {
     this.setState({ blockRunTest: true }); // to block the button
     this.props.updateProgress({ showProgress: true });
     const self = this;
-    dataFetch('/api/perf/load-test-prefs', {
+    dataFetch('/api/user/prefs/perf', {
```

Comment:
Incorrect endpoint? Extra `/perf`.

![image](https://user-images.githubusercontent.com/44001539/126040134-f742bedd-0938-4c35-b740-52c222fc1ff5.png)

Reply:
Ooops, rookie mistake. I'll resolve this.

Reply:
This is the proposed endpoint, actually.

Reply:
@dhruv0000 are you running Istio-adapter locally??